dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -840,6 +840,664 @@
+        },
+        "value": {
+          "type": "number",
+          "format": "double",
+          "description": "The money amount captured in the extracted entity"
+        },
+        "unit": {
+          "type": "string",
+          "description": "The unit of the amount captured in the extracted entity"
+        }
+      },
+      "required": [
+        "value",
+        "unit"
+      ]
+    },
+    "BooleanResolution": {
+      "description": "A resolution for boolean expressions",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "BooleanResolution",
+      "properties": {
+        "value": {
+          "type": "boolean"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DateTimeResolution": {
+      "description": "A resolution for datetime entity instances.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "DateTimeResolution",
+      "properties": {
+        "timex": {
+          "$ref": "#/definitions/TimeExpression"
+        },
+        "dateTimeSubKind": {
+          "type": "string",
+          "enum": [
+            "Time",
+            "Date",
+            "DateTime",
+            "Duration",
+            "Set"
+          ],
+          "x-ms-enum": {
+            "name": "DateTimeSubKind",
+            "modelAsString": true
+          },
+          "description": "The DateTime SubKind"
+        },
+        "value": {
+          "type": "string",
+          "description": "The actual time that the extracted text denote."
+        },
+        "modifier": {
+          "$ref": "#/definitions/TemporalModifier"
+        }
+      },
+      "required": [
+        "timex",
+        "dateTimeSubKind",
+        "value"
+      ]
+    },
+    "NumberResolution": {
+      "description": "A resolution for numeric entity instances.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "NumberResolution",
+      "properties": {
+        "numberKind": {
+          "type": "string",
+          "enum": [
+            "Integer",
+            "Decimal",
+            "Power",
+            "Fraction",
+            "Percent",
+            "Unspecified"
+          ],
+          "x-ms-enum": {
+            "name": "NumberKind",
+            "modelAsString": true
+          },
+          "description": "The type of the extracted number entity."
+        },
+        "value": {
+          "type": "string",
```

Comment:
should this be updated to double across specs? @joseharriaga 

Also, we should update 2022-05-01 while we're making this change.

Reply:
I recall that @bidisha-c has this other PR where she does this, but I agree that it's probably simpler to just bundle the changes as part of this one.

https://github.com/Azure/azure-rest-api-specs/pull/21403/files

+1 on making the changes to 2022-05-01 too.

Reply:
ok.